src/Home.js

```diff
@@ -1,17 +1,127 @@
+            subscription = DataStore.observe(Todo).subscribe((msg) => {
+                onQuery();
+            });
+        }
+
+        return function cleanup() {
+            subscription.unsubscribe();
+        }
+    }, []);
+
+    async function setComplete(updateValue, todo) {
+        await DataStore.save(
+            Todo.copyOf(todo, updated => {
+                updated.isComplete = updateValue
+            })
+        );
+    }
+
+    const todoItem = ({ item }) => (
+        <View style={styles.todoContainer}>
+            <View>
+                <Text style={styles.todoHeading} >{item.name}</Text>
+                <Text style={styles.todoDescription} >{item.description}</Text>
+            </View>
+            <TouchableOpacity
+                style={[styles.checkbox, item.isComplete && styles.selectedCheckbox]}
+                onPress={() => {
+                    setComplete(!item.isComplete, item)
+                }}
+            >
+                <Text style={[styles.checkboxText, item.isComplete && styles.selectedCheckboxText]}> {(item.isComplete) ? "✓" : ""} </Text>
+            </TouchableOpacity>
+        </View >
+    );
+
     return (
-        <View style={styles.container}>
-            <Text>Todo App</Text>
-        </View>
+        <SafeAreaView style={styles.container}>
+            <FlatList
+                data={todos}
+                renderItem={todoItem}
+                keyExtractor={item => item.id}
+            />
+        </SafeAreaView>
     );
 }
 
 const styles = StyleSheet.create({
     container: {
-        alignItems: 'center',
-        justifyContent: 'flex-start',
+        flex: 1,
+    },
+    todoContainer: {
+        marginVertical: 5,
+        marginHorizontal: 10,
+        flexDirection: "row",
+        justifyContent: "space-between",
+        backgroundColor: "white",
+        shadowColor: "#000000",
+        shadowOpacity: 0.3,
+        shadowRadius: 2,
+        shadowOffset: {
+            height: 1,
+            width: 1
+        }
+    },
+    todoHeading: {
+        fontSize: 20,
+        fontWeight: "600",
+        padding: 5,
+    },
+    todoDescription: {
+        padding: 5,
+    },
+    checkbox: {
+        alignSelf: "center",
+        alignItems: "center",
+        borderWidth: 2,
+        height: 20,
+        width: 20,
+        margin: 10,
+        borderRadius: 2,
+    },
+    checkboxText: {
+        fontSize: 13,
+        fontWeight: "700",
+        alignSelf: 'center',
+    },
+    selectedCheckbox: {
+        backgroundColor: "black"
+    },
+    selectedCheckboxText: {
+        color: "white"
+    },
+    buttonText: {
+        fontSize: 16,
+        fontWeight: "600",
+        padding: 5,
+        alignSelf: "center",
+        color: "#fff"
+    },
+    button: {
+        backgroundColor: "#4696ec",
+        width: 150,
+        alignSelf: "center",
+        borderRadius: 25,
+        padding: 10,
+        margin: 10,
+        shadowOpacity: 0.3,
```

Comment:
Don't see either of these styles being used in this PR, can we wait to add them until they are needed :)

Reply:
Forgot to remove them. I'll update that as well.